src/uwtools/tests/test_cli.py

```diff
@@ -161,6 +161,42 @@ def test__add_subparser_template_translate(subparsers):
     assert subparsers.choices[STR.translate]
 
 
+def test__dispatch_invoke():
+    cycle = dt.datetime.now()
+    args: dict = {
+        "module": "testdriver",
+        "classname": "TestDriver",
+        "schema_file": "path/to/testdriver.jsonschema",
+        "module_dir": "path/to/dir",
+        "batch": True,
+        "config_file": "/path/to/config",
+        "cycle": cycle,
+        "leadtime": None,
+        "dry_run": False,
+        "graph_file": None,
+        "key_path": ["foo", "bar"],
+        "task": "eighty_eight",
+        "stdin_ok": True,
+    }
+    with patch.object(cli.uwtools.api.driver, "execute") as execute:
+        cli._dispatch_invoke(args=args)
+        execute.assert_called_once_with(
+            classname="TestDriver",
+            module="testdriver",
+            task="eighty_eight",
+            schema_file="path/to/testdriver.jsonschema",
+            key_path=["foo", "bar"],
+            dry_run=False,
+            config="/path/to/config",
+            module_dir="path/to/dir",
+            graph_file=None,
+            cycle=cycle,
+            leadtime=None,
+            batch=True,
+            stdin_ok=True,
+        )
```

Comment:
```suggestion
    args: dict = {
        "module": "testdriver",
        "classname": "TestDriver",
        "schema_file": "/path/to/testdriver.jsonschema",
        "module_dir": "/path/to/dir",
        "batch": True,
        "config_file": "/path/to/config",
        "cycle": cycle,
        "leadtime": None,
        "dry_run": False,
        "graph_file": None,
        "key_path": ["foo", "bar"],
        "task": "eighty_eight",
        "stdin_ok": True,
    }
    with patch.object(cli.uwtools.api.driver, "execute") as execute:
        cli._dispatch_invoke(args=args)
        execute.assert_called_once_with(
            classname="TestDriver",
            module="testdriver",
            task="eighty_eight",
            schema_file="/path/to/testdriver.jsonschema",
            key_path=["foo", "bar"],
            dry_run=False,
            config="/path/to/config",
            module_dir="/path/to/dir",
            graph_file=None,
            cycle=cycle,
            leadtime=None,
            batch=True,
            stdin_ok=True,
        )
```
Added leading `/` to paths for consistency.